Challenge3/src/agent.ts

```diff
@@ -0,0 +1,54 @@
+import { Finding, HandleBlock, BlockEvent, getEthersProvider, Initialize, ethers, getAlerts } from "forta-agent";
+import { L1_DAI_TOKEN_ADDRESS, L1_ESCROW_ABI, L1_ARB_ESCROW_ADDRESS, L1_OPT_ESCROW_ADDRESS } from "./constants";
+import { Contract } from "@ethersproject/contracts";
+import { getL1DaiLocked, getL2DaiSupply, GetAlertsFunction } from "./utils";
+import { l1Finding, l2Finding } from "./findings";
+
+let chainId: number;
+let l1OptEscrowBalance: ethers.BigNumber = ethers.BigNumber.from(0);
+let l1ArbEscrowBalance: ethers.BigNumber = ethers.BigNumber.from(0);
+let newL1OptEscrowBalance: ethers.BigNumber;
+let newL1ArbEscrowBalance: ethers.BigNumber;
```

Comment:
Declaring these two globally is redundant.
You can declare e.g. `const currentL1OptEscrowBalance` the moment `await l1DaiContract.balanceOf..` is called